### PR TITLE
delete user disable endpoint add config

### PR DESCRIPTION
**Pull request by MariyaSin**

### CODE REVIEW

**Review by borovHokage — APPROVED** [review]

**Review by VLiamin — APPROVED** [review]

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 1]
```diff
@@ -26,313 +23,316 @@
...
+            userFirstName: dbUser.FirstName,
+            userEmail: email.Value,
+            userId: dbUser.Id.ToString(),
+            userPassword: password);
+
+        var emailRequest = ISendEmailRequest.CreateObj(null, senderId, email.Value, templateLanguage, templateType, templateValues);
+
+        IOperationResult<bool> rcSendEmailResponse = _rcSendEmail
+            .GetResponse<IOperationResult<bool>>(emailRequest, timeout: RequestTimeout.Default)
+            .Result
```
Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:188]: same

File: src/UserService.Business/Commands/User/CreateUserCommand.cs [hunk 2]
```diff
@@ -26,313 +23,316 @@
...
-                }
-            }
-            catch (Exception exc)
-            {
-                _logger.LogError(exc, "Can not send email to '{Email}'", email.Value);
-
-                errors.Add(errorMessage);
-            }
+          _logger.LogWarning(
+              "Errors while sending email to '{Email}':\n {Errors}",
```
Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:194]: same

Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:194]: same

File: src/UserService.Business/Commands/User/CreateUserCommand.cs (hunk 1 above)
Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:188]: same

Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:188]: same

File: src/UserService.Business/Commands/User/CreateUserCommand.cs (hunk 2 above)
Comment (BelehovEgor) [inline src/UserService.Business/Commands/User/CreateUserCommand.cs:194]: same

File: src/UserService.Models.Dto/Requests/User/Filters/GetUserFilter.cs [hunk 3]
```diff
@@ -3,60 +3,60 @@
...
-        public bool IsIncludeRole => IncludeRole.HasValue && IncludeRole.Value;
-        public bool IsIncludeAchievements => IncludeAchievements.HasValue && IncludeAchievements.Value;
-        public bool IsIncludeCertificates => IncludeCertificates.HasValue && IncludeCertificates.Value;
-        public bool IsIncludeCommunications => IncludeCommunications.HasValue && IncludeCommunications.Value;
-        public bool IsIncludeImages => IncludeImages.HasValue && IncludeImages.Value;
-        public bool IsIncludeSkills => IncludeSkills.HasValue && IncludeSkills.Value;
-        public bool IsIncludeEducation => IncludeEducations.HasValue && IncludeEducations.Value;
-        public bool IsIncludeProjects => IncludeProjects.HasValue && IncludeProjects.Value;
-    }
+    public bool IsIncludeDepartment => IncludeDepartment;
```
Comment (BelehovEgor) [inline src/UserService.Models.Dto/Requests/User/Filters/GetUserFilter.cs:50]: delete it

**Review by BelehovEgor — APPROVED** [review]

**Review by ndoriomedov — APPROVED** [review]

**Pull request merged**